components/meta-packages/install-types/includes/minimal

```diff
@@ -133,8 +133,7 @@ depend type=require fmri=package/pkg
 depend type=require fmri=package/svr4
 depend type=require fmri=release/name
 depend type=require fmri=release/notices
-depend type=require fmri=runtime/perl-534
-depend type=require fmri=runtime/perl-534/module/sun-solaris
+depend type=require fmri=runtime/perl
 depend type=require fmri=service/fault-management
```

Comment:
How will sun-solaris be installed then?

Reply:
Both `SUNWcs` and `system/extended-system-utilities` requires `sun-solaris`, so I expect the package will be installed automatically to satisfy dependencies.  But, honestly, I do not know how exactly this installation related stuff works so I could be easily wrong and something will fail during the installation.

Similarly I assume the `runtime/perl` meta-package will automatically bring in the right version of `runtime/perl-XXX` based on dependencies of `runtime/perl`.

If this my assumption is not correct, then we cannot do this, obviously.